src/main/java/com/fcgl/madrid/forum/service/CommentService.java

```diff
@@ -45,13 +54,35 @@ public CommentService(CommentRepository commentRepository) {
                     commentRequest.getUserId()
             );
             commentRepository.save(comment);
-            return new ResponseEntity<InternalStatus>(InternalStatus.OK, HttpStatus.OK);
         }
         catch (TransactionSystemException e) {
             return handleParamException(e);
         }
+
+        boolean success = updatePostCommentCount(commentRequest.getPost(), new Integer(1));
+        if (!success) {
+            logger.warn(new String("UPDATE POST COMMENT COUNT FAILED FOR POST: " + Long.toString(commentRequest.getPost().getId())));
+        }
+        return new ResponseEntity<InternalStatus>(InternalStatus.OK, HttpStatus.OK);
     }
 
+    /**
+     * Adds a count to the number of comments. Follows the "eventually consistent strategy".
```

Comment:
Okay, we still need to create the process to make commentSize eventually consistent. We can do this is a separate PR. Basically whenever this logger is set, we will eventually want to update the `commentSize` to the actual size.